flink-end-to-end-tests/test-scripts/test_datastream_walkthroughs.sh

```diff
@@ -35,6 +35,7 @@ mvn archetype:generate                                          \
     -DarchetypeGroupId=org.apache.flink                         \
     -DarchetypeArtifactId=flink-walkthrough-datastream-${TEST_TYPE}  \
     -DarchetypeVersion=${FLINK_VERSION}                         \
+    -DarchetypeCatalog=local                                    \
```

Comment:
did you double-check whether this is still necessary?

Reply:
I'll check

Reply:
Note that I'm not against having it (in fact it would've made debugging this a lot faster!), but then we should do the same for the quickstarts.

Reply:
Not needed, see https://travis-ci.com/flink-ci/flink/builds/144510365